packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -545,6 +555,8 @@ const buildFormSchema = (parentKey: string, response: Record<string, any>, initi
     initialFormSchema[field] = { id: field, model: key } // each field's key will be set as the id
     initialFormSchema[field].type = scheme.type === 'boolean' ? 'checkbox' : 'input'
     initialFormSchema[field].required = scheme.required
+    initialFormSchema[field].values = scheme.values
```

Comment:
Question: what is `values`?

Reply:
Good question. This is to pass the `values` attribute on a `map` field to the field in the generated schema which allows the autofill providers to consume the attributes inside.

For example, with the `http-log` plugin, we have the following schema for the `headers` field:

```json
{
  "headers": {
    "description": "...",
    "type": "map",
    "values": {
      "referenceable": true,
      "type": "string"
    },
    "keys": "..."
  }
}
```

Since we seem to only have `values` for `map` fields now, I've refactored the code and added some comments to ensure this is easy to understand.